src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
 enchantment.openblocks.flimflam=Blödsinn
 
 openblocks.gui.blocks=Blöcke
-openblocks.gui.items=Items
+openblocks.gui.items=Gegenstände
 openblocks.gui.misc=Verschiedenes
-openblocks.gui.changelogs=Changelogs
+openblocks.gui.changelogs=Änderungsprotokoll
 
-openblocks.gui.features=Features
-openblocks.gui.bugfixes=Bugfixes
-openblocks.gui.tweaks=Tweaks
+openblocks.gui.features=Eigenschaften
+openblocks.gui.bugfixes=Fehlerbehebung
+openblocks.gui.tweaks=Optimierungen
 
 openblocks.gui.luggage=Reisegepäck
 openblocks.gui.sprinkler=Rasensprenger
 openblocks.gui.vacuumhopper=Vakuumtrichter
 openblocks.gui.bigbutton=Großer Knopf
 openblocks.gui.xpbottler=XP-Abfüllgerät
-openblocks.gui.autoanvil=Auto-Amboss
-openblocks.gui.autoeject=Auto-Auswerfen
-openblocks.gui.autoextract=Auto-Extrahieren
-openblocks.gui.autodrink=Auto-Trinken
-openblocks.gui.autoenchantmenttable=Auto-Zaubertisch
+openblocks.gui.autoanvil=Automatischer Amboss
+openblocks.gui.autoeject=Automatischer Auswerfer
+openblocks.gui.autoextract=Automatischer Extrahierer
+openblocks.gui.autodrink=Automatisches Trinkgerät
+openblocks.gui.autoenchantmenttable=Automatischer Zaubertisch
 openblocks.gui.donationstation=Spendenstation
-openblocks.gui.itemdropper=Itemspender
+openblocks.gui.itemdropper=Gegenstandspender
 openblocks.gui.blockplacer=Blockplatzierer
-openblocks.gui.paintmixer=Farbmixer
+openblocks.gui.paintmixer=Farbenmischer
 openblocks.gui.drawingtable=Zeichentisch
-openblocks.gui.digitalfuse=Digitalzünder
-openblocks.gui.xp_outputs=XP-Abgabe
-openblocks.gui.item_outputs=Itemabgabe
-openblocks.gui.limit=Limit
+openblocks.gui.digitalfuse=Digitaler Zünder
+openblocks.gui.xp_outputs=Erfahrungspunkten-Abgabe
+openblocks.gui.item_outputs=Gegenstands-Abgabe
+openblocks.gui.limit=Grenze
 openblocks.gui.available_power=Verfügbare Energie: %d
-openblocks.gui.save_folder=Sichere Ordner
+openblocks.gui.save_folder=Ordner sichern
 
 openblocks.gui.drawingtable.stencils=Schablonen
 openblocks.gui.drawingtable.glyphs=Glyphen
 openblocks.gui.drawingtable.print=Druck
 
 openblocks.gui.welcome.title=Willkommen!
-openblocks.gui.welcome.content=OpenBlocks ist eine neue Open-Source-Mod, die verschiedene Ideen in Minecraft einbringt, inklusive Gleitschirme, Aufzüge, Tanks, Gräber, Farben, Kräne, flüssige XP, Kartographen, Trophäen und mehr!
+openblocks.gui.welcome.content=OpenBlocks ist eine neue freie Modifikation, die verschiedene Ideen in Minecraft einbringt, inklusive Gleitschirme, Aufzüge, Tanks, Gräber, Farben, Kräne, flüssige Erfahrungspunkte, Kartographen, Trophäen und mehr!
```

Comment:
It should stay "Mod".

Reply:
"Mod" is in the dictionary, but it didn't have the same meaning as modification :D 
https://www.duden.de/rechtschreibung/Mod

Reply:
But... It was obvious in this context. Are you not translating without context in mind?

Reply:
Is there a problem if we use the word "Modifikation"?

Reply:
It is inconsistent, and mostly just sounds bad. "Mod" is a known word and should be used.

Reply:
But no, it is not correct.

Reply:
Still not, however.